src/objects.jl

```diff
@@ -22,10 +22,16 @@ end
 # Object (group, named datatype, or dataset) open
 function h5object(obj_id::API.hid_t, parent)
     obj_type = API.h5i_get_type(obj_id)
-    obj_type == API.H5I_GROUP ? Group(obj_id, file(parent)) :
-    obj_type == API.H5I_DATATYPE ? Datatype(obj_id, file(parent)) :
-    obj_type == API.H5I_DATASET ? Dataset(obj_id, file(parent)) :
-    error("Invalid object type for path ", path)
+    obj = if obj_type == API.H5I_GROUP
+        Group(obj_id, file(parent))
+    elseif obj_type == API.H5I_DATATYPE
+        Datatype(obj_id, file(parent))
+    elseif obj_type == API.H5I_DATASET
+        Dataset(obj_id, file(parent))
+    else
+        error("Invalid object type for path ", path)
+    end
+    obj
```

Comment:
```suggestion
    obj_type == API.H5I_GROUP ? Group(obj_id, file(parent)) :
    obj_type == API.H5I_DATATYPE ? Datatype(obj_id, file(parent)) :
    obj_type == API.H5I_DATASET ? Dataset(obj_id, file(parent)) :
    error("Invalid object type for path ", path)
```

This is an unrelated stylistic change. If we are going to do this, we should do this globally rather than just right here. properties.jl uses the existing style a lot.

Reply:
Probably leftovers from developing. Restored.